### PR TITLE
Vxlan upgrade fix

### PR DESCRIPTION
**Pull request by leodotcloud**
https://github.com/rancher/rancher/issues/8242

### CODE REVIEW

File: infra-templates/vxlan/10/rancher-compose.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@
   name: "Rancher VXLAN"
   version: "v0.1.2"
```
Comment (deniseschannon) [inline infra-templates/vxlan/10/rancher-compose.yml]: we should bump the version to v0.2.0, if these are fresh install only. you'll need to also bump up the version in the rancher-compose.yml

File: infra-templates/vxlan/9/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+## VXLAN Networking
+
+Rancher networking plugin using VXLAN overlay.
+
+### Open Ports
+
+Traffic to and from hosts requires UDP port `4789` to be open.
+
+### Upgrade options
```
Comment (deniseschannon) [inline infra-templates/vxlan/9/README.md]: Would we want to give out options of deleting the infra service and starting a new vxlan service? would that work for people who want to keep the same environment?

Comment (leodotcloud) [reply infra-templates/vxlan/9/README.md]: Due to dependencies check in Cattle, deleting "vxlan" infra service is not possible without deleting all the existing stacks, scheduler and healthcheck. At this point it's almost similar to creating a new environment.

**Comment by leodotcloud** [comment]
Tests:
Before: Setup with 1.5.7, vxlan environment running with vxlan: 0.0.7
Upgrade to 1.6.6-rc4.
After: No upgrade option available for vxlan stack in the old vxlan environment.
Now if I create a new vxlan2 environment, I am able to see the new 0.2.0 template.


screen shot 2017-07-27 at 2 26 27 pm

screen shot 2017-07-27 at 2 22 22 pm

screen shot 2017-07-27 at 2 24 04 pm

**Pull request merged**